code/game/objects/items/weapons/storage/marshal_kits.dm

```diff
@@ -86,7 +86,7 @@
 	desc = "The standard Marshal box kit containing a Viper DMR. A marksman carbine for a true marksman."
 
 	populate_contents()
-		new /obj/item/gun/projectile/automatic/mamba/viper
+		new /obj/item/gun/projectile/automatic/mamba/viper (src)
```

Comment:
Should there be a space between viper and (src)?

Reply:
Fixed